src/components/Filters/Filters.tsx

```diff
@@ -169,17 +180,27 @@ export const Filters = ({ filters, setFilters, pages }: Props) => (
         </select>
       </div>
     </div>
-    <div className={paginationWrapper}>
-      <Pagination
-        pages={pages}
-        page={filters.page}
-        setPage={(page: number) => {
-          setFilters({
-            ...filters,
-            page: page
-          });
-        }}
-      />
-    </div>
+    {/* Don't show pagination if there's less than 2 pages, no point */}
+    {filters.page < 2 && (
+      <div className={paginationWrapper}>
+        <div className={paginationSelections}>
+          <Pagination
+            pages={pages}
+            page={filters.page}
+            setPage={(page: number) => {
+              setFilters({
+                ...filters,
+                page: page
+              });
+            }}
+          />
+        </div>
+        <div className={paginationText}>
+          {commentCount &&
+            `Displaying ${filters.pageSize *
+              filters.page} of ${commentCount} comments`}
+        </div>
```

Comment:
There are no designs for what appears at the footer of the list but assuming we show pagination in the same way, with this text, then we might want to think about pulling this code out.

Might also need to guard against multiply by zero errors and other edge cases

Reply:
technically the `filters.page < 2` should avoid any multiply by zero errors